### PR TITLE
Add `credits` tile in `AboutPage`

### LINKED ISSUE #183: Add `Credits` section in `AboutPage`

### PR DESCRIPTION
**Pull request by prestosole**
.

### CODE REVIEW

**Comment by triallax** [comment]
> Only thing left is the color of the hyper links.

You didn't do that yet.

**Comment by prestosole** [comment]
You didn't tell me how to fix it yet!

**Comment by triallax** [comment]
i guess you can change `LinkTextStyle` to a function which takes a `BuildContext` as an argument.

**Comment by prestosole** [comment]
Resolved!

File: packages/clima_ui/lib/widgets/settings/link_text_span.dart [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/gestures.dart';
```
Comment (triallax) [inline packages/clima_ui/lib/widgets/settings/link_text_span.dart]: Don't put this under `settings`; put it directly under `widgets`. There's nothing specific to the settings page here.

**Review by triallax — CHANGES_REQUESTED** [review]

File: packages/clima_ui/lib/widgets/settings/link_text_span.dart (hunk 1 above)
Comment (prestosole) [reply packages/clima_ui/lib/widgets/settings/link_text_span.dart]: Do it yourself.

Comment (prestosole) [reply packages/clima_ui/lib/widgets/settings/link_text_span.dart]: Oh, you actually changed it yourself!

**Review by triallax — APPROVED** [review]

**Pull request merged**